src/platform/packages/shared/kbn-alerts-ui-shared/src/alerts_search_bar/index.test.tsx

```diff
@@ -98,199 +126,419 @@ describe('AlertsSearchBar', () => {
 
-  it('calls the unifiedSearchBar correctly for NON security rule types', async () => {
-    render(
-      <AlertsSearchBar
-        rangeFrom="now/d"
-        rangeTo="now/d"
-        query=""
-        onQuerySubmit={jest.fn()}
-        toasts={toastsMock}
-        http={httpMock}
-        dataService={mockDataPlugin}
-        appName={'test'}
-        onFiltersUpdated={jest.fn()}
-        unifiedSearchBar={unifiedSearchBarMock}
-        ruleTypeIds={['.esQuery']}
-      />
-    );
+    it('calls the unifiedSearchBar with correct index patters without rule types', async () => {
+      render(
+        <AlertsSearchBar
+          rangeFrom="now/d"
+          rangeTo="now/d"
+          query=""
+          onQuerySubmit={jest.fn()}
+          toasts={toastsMock}
+          http={httpMock}
+          dataService={mockDataPlugin}
+          appName={'test'}
+          onFiltersUpdated={jest.fn()}
+          unifiedSearchBar={unifiedSearchBarMock}
+        />
+      );
 
-    await waitFor(() => {
-      expect(unifiedSearchBarMock).toHaveBeenCalledWith(
-        expect.objectContaining({
-          suggestionsAbstraction: { type: 'alerts', fields: {} },
-        })
+      await waitFor(() => {
+        expect(unifiedSearchBarMock).toHaveBeenCalledWith(
+          expect.objectContaining({
+            indexPatterns: [
+              {
+                fields: [
+                  { aggregatable: true, name: 'event.action', searchable: true, type: 'string' },
+                ],
+                title: '.alerts-*',
+              },
+            ],
+          })
+        );
+      });
+    });
+
+    it('calls the unifiedSearchBar with correct index patters without data views', async () => {
+      jest.mocked(useAlertsDataView).mockReturnValue({
+        isLoading: false,
+        dataView: undefined,
+      });
+
+      render(
+        <AlertsSearchBar
+          rangeFrom="now/d"
+          rangeTo="now/d"
+          query=""
+          onQuerySubmit={jest.fn()}
+          toasts={toastsMock}
+          http={httpMock}
+          dataService={mockDataPlugin}
+          appName={'test'}
+          onFiltersUpdated={jest.fn()}
+          unifiedSearchBar={unifiedSearchBarMock}
+        />
       );
+
+      await waitFor(() => {
+        expect(unifiedSearchBarMock).toHaveBeenCalledWith(
+          expect.objectContaining({
+            indexPatterns: [],
+          })
+        );
+      });
     });
   });
 
-  it('calls the unifiedSearchBar with correct index patters', async () => {
-    render(
-      <AlertsSearchBar
-        rangeFrom="now/d"
-        rangeTo="now/d"
-        query=""
-        onQuerySubmit={jest.fn()}
-        toasts={toastsMock}
-        http={httpMock}
-        dataService={mockDataPlugin}
-        appName={'test'}
-        onFiltersUpdated={jest.fn()}
-        unifiedSearchBar={unifiedSearchBarMock}
-        ruleTypeIds={['.esQuery', 'apm.anomaly']}
-      />
-    );
+  describe('With useAlertsNewApiDataView', () => {
```

Comment:
Do you mean this?
```suggestion
  describe('With enableNewAPIForFields', () => {
```

Reply:
Added [test](https://github.com/elastic/kibana/pull/227888/commits/83557b78237a0293b317a68f95abdd6b82411122#diff-29a5ad93b8c227f254a34b1bcbce679b7422c883894df0f655c447c296d78661) to verify `useFetchAlertsFieldsWithNewApi` is called with `enabled: true`
` /fields` url is verified in `useFetchAlertsFieldsWithNewApi` tests.